lib/server/feed.js

```diff
@@ -17,7 +17,7 @@ const siteConfig = require(CWD + '/siteConfig.js');
 
 const blogFolder = path.resolve('../blog/');
 const blogRootURL = siteConfig.url + '/blog';
-const jestImage = siteConfig.url + siteConfig.headerIcon;
+const jestImage = siteConfig.url + '/' + siteConfig.headerIcon;
```

Comment:
I probably should change the name of this sometime from `jestImage` to `docusaurusImage` or something :)